tutorials/shaders/shader_reference/shading_language.rst

```diff
@@ -637,13 +637,25 @@ Example below:
         result = a + b;
     }
 
-.. note::
+Starting off Godot 4.4 you can now define multiple functions with the same name but different
+arguments (**overloading**). An implicit constant conversion (e.g. `1` -> `1.0`) will be disabled
+on function call for such cases to prevent ambiguity.
```

Comment:
```suggestion
Function overloading is supported. You can define multiple functions with the same
name, but different arguments. Note that `implicit casting <Casting_>`_ in function
calls is not allowed, such as from ``int`` to ``float`` (``1`` to ``1.0``).
```

In most cases the docs do not need to specify the first version in which a feature is supported, since it's only relevant for the first few versions after a change (and only if you re-read the docs page on each release).

In Godot 4.3, implicit casting is already forbidden for GDShader, including for function calls:
![Godot_v4 3-stable_win64_QCagaFSYj7](https://github.com/user-attachments/assets/3054f6dc-15b5-4353-87bd-5378f3da9c52)

Also revised the paragraph for grammar.